### ISSUE TITLE
Enable auto-completion

### ISSUE THREAD

**Issue by supermamie** [NimaKhajehpour/GuessThatPokemon#10]
This should be enabled or disabled in the main conf.
Some pokemons, I know their name phonetically, but the real script is nearly impossible to guess in 3 times.

And for some of thems, I do not have the right caracter, like Nidoran♂

Tge auto-completion should, or not, be linked to the generation used.

**Comment by NimaKhajehpour** [comment]
Yeah the special names are a littke challenging to implement. I guess one of the ways is to add auto completion

**Comment by NimaKhajehpour** [comment]
i am thinking of a way to circumvent the issue with special characters and will add that, as for auto completion i think it will just make the game too easy. but will fix the problem with the special characters
